packages/table/src/V11Adapter/V11Adapter.tsx

```diff
@@ -158,18 +191,29 @@ const V11Adapter = <T extends LGRowData>({
                 <ExpandedContent row={row} />
               )}
               {row.subRows &&
-                row.subRows.map(subRow => (
-                  <Row key={subRow.id} row={subRow}>
-                    {subRow.getVisibleCells().map(subRowCell => {
-                      return (
-                        <Cell key={subRowCell.id}>
-                          {/* @ts-expect-error subRow.original returns the object in the user's defined shape, and should be string indexable */}
-                          {subRow.original[subRowCell.column.id]()}
-                        </Cell>
-                      );
-                    })}
-                  </Row>
-                ))}
+                row.subRows.map(subRow => {
+                  // @ts-expect-error rowProps is an additional prop passed by the `processData` function
+                  const { children, ...subRowProps } = subRow.original.rowProps;
+                  return (
+                    <Row key={subRow.id} row={subRow} {...subRowProps}>
+                      {subRow.getVisibleCells().map(srCell => {
+                        /* @ts-expect-error subRow.original returns the object in the user's defined shape, and should be string indexable */
+                        const subRowCell = subRow.original[srCell.column.id]();
```

Comment:
Seems there are a lot of issues with the TS type of `row.original` vs how it's used.  We should fix those

Reply:
agree that we should fix htis.